### PR TITLE
Add method for Clear Db to Cache

### PR DESCRIPTION
**Pull request by ekaterinaryzhenkova**

### CODE REVIEW

File: src/Kernel.RedisSupport/Helpers/CacheNotebook.cs [hunk 1]
```diff
@@ -82,6 +82,19 @@ public void Remove(Guid elementId)
...
+  public void Clear(int database)
+  {
+    var keysToRemove = _dictionary
+        .Where(k => k.Value.Any(frame => frame.Database == database))
+        .Select(k => k.Key)
+        .ToList();
+
+    foreach (Guid key in keysToRemove)
+    {
+      _dictionary.TryRemove(key, out _);
```
Comment (SolovyovDmitry) [inline src/Kernel.RedisSupport/Helpers/CacheNotebook.cs]: add logging here and to whole class

File: src/Kernel.RedisSupport/Helpers/CacheNotebook.cs [hunk 2]
```diff
@@ -82,6 +82,19 @@ public void Remove(Guid elementId)
     _dictionary.TryRemove(elementId, out _);
   }
 
+  public void Clear(int database)
+  {
+    var keysToRemove = _dictionary
+        .Where(k => k.Value.Any(frame => frame.Database == database))
```
Comment (SolovyovDmitry) [inline src/Kernel.RedisSupport/Helpers/CacheNotebook.cs]: not 'frame' but 'f'

File: src/Kernel.RedisSupport/Helpers/GlobalCacheRepository.cs [hunk 3]
```diff
@@ -56,6 +56,20 @@ public async Task<bool> RemoveAsync(Guid elementId)
...
+
+    if (!await _redisHelper.RemoveAsync(elements))
+    {
+      return false;
+    }
+
+    _cacheNotebook.Clear(database);
+
+    return true;
+  }
```
Comment (SolovyovDmitry) [inline src/Kernel.RedisSupport/Helpers/GlobalCacheRepository.cs]: same about logging

File: src/Kernel.RedisSupport/Helpers/Interfaces/ICacheNotebook.cs [hunk 4]
```diff
@@ -12,5 +12,6 @@ public interface ICacheNotebook
   IEnumerable<(int database, string key)> GetKeys(Guid elementId);
   IEnumerable<(int database, string key)> GetKeys();
   void Remove(Guid elementId);
+  void Clear(int database);
```
Comment (SolovyovDmitry) [inline src/Kernel.RedisSupport/Helpers/Interfaces/ICacheNotebook.cs:15]: added here but didn't in IGlobalCacheRepository

**Review by SolovyovDmitry — CHANGES_REQUESTED** [review]

File: src/Kernel.RedisSupport/Helpers/Interfaces/ICacheNotebook.cs (hunk 4 above)
Comment (ekaterinaryzhenkova) [reply src/Kernel.RedisSupport/Helpers/Interfaces/ICacheNotebook.cs:15]: fixed

File: src/Kernel.RedisSupport/Helpers/GlobalCacheRepository.cs (hunk 3 above)
Comment (ekaterinaryzhenkova) [reply src/Kernel.RedisSupport/Helpers/GlobalCacheRepository.cs]: added

File: src/Kernel.RedisSupport/Helpers/CacheNotebook.cs (hunk 2 above)
Comment (ekaterinaryzhenkova) [reply src/Kernel.RedisSupport/Helpers/CacheNotebook.cs]: fixed

File: src/Kernel.RedisSupport/Helpers/CacheNotebook.cs (hunk 1 above)
Comment (ekaterinaryzhenkova) [reply src/Kernel.RedisSupport/Helpers/CacheNotebook.cs]: added

Comment (SolovyovDmitry) [reply src/Kernel.RedisSupport/Helpers/CacheNotebook.cs]: also add for positive cases

File: src/Kernel.RedisSupport/Helpers/CacheNotebook.cs [hunk 5]
```diff
@@ -2,6 +2,7 @@
 using LT.DigitalOffice.Kernel.RedisSupport.Helpers.Interfaces;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
+using StackExchange.Redis;
```
Comment (SolovyovDmitry) [inline src/Kernel.RedisSupport/Helpers/CacheNotebook.cs:5]: why did it appear here?

**Review by SolovyovDmitry — APPROVED** [review]

**Pull request merged**